pom.xml

```diff
@@ -47,23 +47,25 @@
     </description>
 
     <inceptionYear>2010</inceptionYear>
-    <url>http://www.axonframework.org</url>
+    <url>https://axoniq.io/</url>
     <licenses>
         <license>
             <name>Apache 2.0</name>
-            <url>http://www.apache.org/licenses/LICENSE-2.0</url>
+            <url>https://www.apache.org/licenses/LICENSE-2.0</url>
         </license>
     </licenses>
     <issueManagement>
         <system>GitHub</system>
-        <url>http://issues.axonframework.org</url>
+        <url>https://github.com/AxonFramework/AxonFramework</url>
```

Comment:
I think this should have /issues behind the URL, in order to point to the issues page.

Reply:
Good point! I'll change the URL.